qwalk.py

```diff
@@ -16,7 +16,10 @@ def main() -> None:
         "-u", help="Qumulo API user", default=os.getenv("QUSER") or "admin"
     )
     parser.add_argument(
-        "-p", help="Qumulo API password", default=os.getenv("QPASS") or "admin"
+        "-p", help="Qumulo API password", default=os.getenv("QPASS") or "'Admin123'"
+    )
+    parser.add_argument(
+        "-a", help="Qumulo access token", default=os.getenv("QTOKEN")
```

Comment:
Let's make these mutually exclusive (https://docs.python.org/3/library/argparse.html#mutual-exclusion) since you have the RestClient init prefer the token over the password. Which is reasonable, but I could see confusion if someone gives the script a valid password + invalid token, and it falls to connect

Better to make that situation impossible